sim/global-types.ts

```diff
@@ -1088,7 +1088,7 @@ interface SpeciesData {
 	requiredMove?: string;
 	battleOnly?: string | string[];
 	isGigantamax?: string;
-	inheritsFrom?: string;
+	changesFrom?: string;
```

Comment:
```suggestion
	/**
	 * How to obtain this Pokémon? This is like `prevo` but for forme changes.
	 * For in-battle forme changes, use `battleOnly`. If a Pokémon needs to
	 * HOLD an item to keep the forme it will be in `requiredItem`, but if it only
	 * needs a one-time USE of an item, it will be in `evoItem`.
	 */
	changesFrom?: string;
```

(Can you fill out `evoItem` for me?)

Reply:
Also we should update `FORMES.md` with this change.

Reply:
/**
	 * How to obtain this Pokémon? This is like `prevo` but for forme changes.
	 * If a Pokémon needs to HOLD an item to keep the forme, it will be in
	 * `requiredItem`.
	 */
	changesFrom?: string;